tools/block-generator/run_runner.sh

```diff
@@ -50,10 +50,9 @@ start_postgres
 echo "Starting test runner"
 $(dirname "$0")/block-generator runner \
 	--conduit-binary "$CONDUIT_BINARY" \
-	--report-directory OUTPUT_RUN_RUNNER_TEST \
+	--report-directory ../../tmp/OUTPUT_RUN_RUNNER_TEST \
 	--test-duration 30s \
 	--log-level trace \
 	--postgres-connection-string "host=localhost user=algorand password=algorand dbname=generator_db port=15432 sslmode=disable" \
 	--scenario ${CONFIG} \
-	--db-round ${2:-0} \
-  --genesis-file ${3:-""} \
+  --reset-db
```

Comment:
I decided to keep this script as simple as possible.

Reply:
This PR should also go ahead of #5394 because I'm replicating the script in python so will want to reflect these changes.